src/services/configuration.service.ts

```diff
@@ -1,15 +1,60 @@
 import * as consign from 'consign';
 import { Injectable } from 'injection-js';
+import { CommonService } from '../services/common.service';
 
 @Injectable()
 export class ConfigurationService {
   private config: any = {};
 
-  constructor() {
+  constructor(private common: CommonService) {
     consign({ cwd: process.env.CONFIG }).include('.').into(this.config);
   }
 
-  getFragment(name: string): string {
-    return `assets/${this.config.fragments[ name ]}`;
+  async getFragment(name: string): Promise<string> {
```

Comment:
Die Methoden zum Zugriff auf die Dateien sollten in einen eigenen Service verschoben werden. Im ConfigurationService würde ich nur die Zugriffe auf die Konfiguration erwarten.